src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2877,6 +2877,12 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Error_HC_QR_CODE_ERROR" = "Der QR-Code konnte nicht generiert werden. Sollte der Fehler weiterhin bestehen, kontaktieren Sie die Hotline über App-Informationen -> Technische Hotline.";
 
+"HealthCertificate_Error_invalidSignature_title" = "Signatur ungültig";
+
+"HealthCertificate_Error_invalidSignature_msg" = "Die Signatur des Zertifikates ist ungültig. Das Zertifikat kann nicht als Nachweis verwendet werden und daher nicht importiert werden.";
```

Comment:
In other places we write "Zertifikats", it should be clarified what we should use "Zertifikates" with an "e" or "Zertifikats" without an "e".

@NataliaLemmerth

Reply:
Hi @Ein-Tim , thanks very much for that. Indeed, we had discussed this several times, we will go ahead and haarmonize the use of "Zertifikats" and "Zertifikates".

Reply:
@ChristinaTsigganos will you update the text used here?

Reply:
@ChristinaTsigganos I updated the text as discussed - please have a look.